ppx_version.opam

```diff
@@ -1,5 +1,14 @@
-opam-version: "1.2"
+opam-version: "2.0"
 version: "0.1"
-build: [
-  ["dune" "build" "--only" "src" "--root" "." "-j" jobs "@install"]
+synopsis: ""
+description:
+  "OCaml extension point (ppx) meant to assure the stability of types and their Bin_prot serializations."
+maintainer: "opensource@o1labs.org"
+depends: [
+  "ocaml" {<= "4.07.1"}
+  "ppxlib"
+  "ppx_bin_prot"
+  "core_kernel" {<= "v0.12.3"}
+  "dune" {>= "2.8.5"}
```

Comment:
Why this version? The `dune-project` file indicates that `2.5` is the minimum supported version, and I imagine this supports even older versions.

Reply:
ah, this was added automatically by one of the tool I used (opam-dune-lint or dune-release I think)